README.md

```diff
@@ -1314,6 +1314,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Muiruriscode](https://github.com/Muiruriscode)
   - [Mujahid Ali](https://github.com/MujahidAli954)
   - [Mukhtar Abdulfatai](https://github.com/DevMukhtarr)
+  - [Muktar Aliyu](https://github.com/zenoftech) :heart:
```

Comment:
@zenofTech please can you remove the `:heart:` from the end of line 1317 so it is in the same format as the rest of the file. Thank you.